samples/Owin.Nowin.HelloWorld/NowinServer.cs

```diff
@@ -44,7 +55,7 @@ public void Dispose()
 
         private Task HandleRequest(IDictionary<string, object> env)
         {
-            return _callback(new DefaultHttpContext(new OwinFeatureCollection(env)));
+            return _callback(new OwinFeatureCollection(env));
```

Comment:
Note that this doesn't actually work since `OwinFeatureCollection` is readonly and `RequestServicesContainerMiddleware` throws when trying to add the `IServiceProvidersFeature`. I bypassed this by checking the `IsReadOnly` property on the `HttpContext.Feature` but we might want to see if there's something different we want to do. @Tratcher